src/helpers.ts

```diff
@@ -76,167 +69,37 @@ export async function createAsset(
 	providerUrl: string,
 	config: Config,
 	aquariusInstance: Aquarius,
+	encryptDDO: boolean = true,
 	templateIndex: number = 1,
+	providerFeeToken: string = ZERO_ADDRESS,
+	accessListFactory?: string,
+	allowAccessList?: string,
+	denyAccessList?: string,
 	macOsProviderUrl?: string,
-	encryptDDO: boolean = true,
+	
 ) {
 	const { chainId } = await owner.provider.getNetwork();
-	const nft = new Nft(owner, chainId);
-	const nftFactory = new NftFactory(config.nftFactoryAddress, owner);
-
-	let wrappedSigner
-	let allowListAddress
 	if(templateIndex === 4){
```

Comment:
this should be the template id or the template address... 
template index is something else (and must be calculated on SDK side)
might work, but is not exactly the same

Reply:
the name is misleading, it should be template id, correct, I'll update this.

Reply:
I think this still needs to be updated